roles/dtc/common/templates/ndfc_vrf_lite.j2

```diff
@@ -1,6 +1,6 @@
 {% if item.ospf is defined and item.ospf %}
-{% include 'ndfc_vrf_lite_ospf.j2' %}
+  {% include 'ndfc_vrf_lite_ospf.j2' %}
+{% endif %}
+{% if (item.bgp is defined and item.bgp) or switch_item.bgp_peers or switch_item.bgp %}
+  {% include 'ndfc_vrf_lite_ebgp.j2' %}
```

Comment:
Same question as from OSPF; how does the indentation of line 2 here make the rendered configuration look? Is the indentation impactful at all as we've left-aligned Jinja2 in most other files if not mistaken.

Reply:
@mtarking - There is no impact.  Other jinja templates have indented formatting for readability, without impact to code/cli alignment.  The jinja commands and indentation are separate from the raw config indentation (see: https://github.com/netascode/ansible-dc-vxlan/blob/develop/roles/dtc/common/templates/ndfc_vrf_lite_ospf.j2) 

Here are the results from this PR with the indented jinja template and produces proper output.

```yaml
- switch:
    - ip: 192.168.112.7
      policies:
        - create_additional_policy: False
          description: nac_myvrf_50001_vrf_lite_netascode4-ospf-bl2
          name: switch_freeform
          policy_vars:
            CONF: |-
              ! This NDFC VRF-Lite eBGP config data structure is auto-generated
              ! DO NOT EDIT MANUALLY
              !
              !
              !
              vrf context myvrf_50001
              router bgp 65003
                vrf myvrf_50001
                  bestpath as-path multipath-relax
                  graceful-restart
                  graceful-restart-helper
                  neighbor 10.31.95.95
                    remote-as 65099
                    ebgp-multihop 2
                    address-family ipv4 unicast
                      route-map extcon-rmap-filter-v6 out
                      allowas-in 3
              !
```